src/core/connectors.ts

```diff
@@ -2341,10 +2341,10 @@ export default <ConnectorMeta[]>[
 		id: 'fmspins',
 	},
 	{
-		label: 'vk-save',
-		matches: ['*://*.vk-save.com/*'],
-		js: 'vk-save.js',
-		id: 'vk-save',
+		label: 'kissvk',
+		matches: ['*://*.kissvk.me/*'],
+		js: 'kissvk.js',
+		id: 'kissvk',
```

Comment:
@yayuyokitano should this be changed if the site is technically the same? Won't it mean any config will be lost?

Reply:
@inverse Hello. They changed their url and name back to vk-save.com. Should I close the pull request?

Reply:
Changing the ID would mean any manual changes etc under the old site would be lost. let wait to see what others suggest...